packages/blocks/src/database-block/components/row-container.ts

```diff
@@ -56,7 +56,7 @@ export function DataBaseRowContainer(
         border-right: 1px solid var(--affine-border-color);
         transform: translateX(0);
       }
-      .affine-database-block-row-cell-content > .affine-block-element {
+      .affine-database-block-row-cell-content > affine-paragraph {
         width: 100%;
```

Comment:
@zqran I think it would be better to use the `.affine-block-element` class here for better compatibility.
Currently, users can drag and drop any block (except database) into the database.

Reply:
Get! 

But one thing needs to be mentioned, the title column should only be a block of type `text`.

Others, like code, image, quote, database, etc., should not appear in the title column.

Reply:
> Others, like code, image, quote, database, etc., should not appear in the title column.

Need to be clear here. 
/cc @doodlewind

Reply:
We can add a filter to only allow text block to be added to the database.

Reply:
Now there is a schema validator for blocks. See #2291.

Reply:
So do we need to filter dragging blocks?